sdk/cosmos/azure-spring-data-cosmos-core/README.md

```diff
@@ -64,7 +64,7 @@ public class MyItem {
     String _etag;
 }
 ```
-- Supports [Azure Cosmos DB partition](https://docs.microsoft.com/azure/cosmos-db/partition-data). To specify a field of domain class to be partition key field, just annotate it with `@PartitionKey`. When you do CRUD operation, pls specify your partition value. For more sample on partition CRUD, pls refer to [test here](./src/test/java/com/azure/spring/data/cosmos/repository/integration/AddressRepositoryIT.java)
+- Supports [Azure Cosmos DB partition](https://docs.microsoft.com/azure/cosmos-db/partition-data). To specify a field of domain class to be partition key field, just annotate it with `@PartitionKey`. When you do CRUD operation, pls specify your partition value. For more sample on partition CRUD, pls refer to [test here](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/cosmos/azure-spring-data-cosmos-test/src/test/java/com/azure/spring/data/cosmos/repository/integration/AddressRepositoryIT.java)
```

Comment:
You can use the relative link rather than a fully qualified one?

Reply:
Yeah but eventually we will be changing this to the full link to the master

Reply:
@KarishmaGhiya  - please update your PR with the latest master, I recently fixed all readme issues with azure-spring-data-cosmos-core and other azure-spring-data-cosmos modules. Here was my PR: https://github.com/Azure/azure-sdk-for-java/pull/13985